compiler/tests/utils.rs

```diff
@@ -225,6 +280,36 @@ pub fn deploy_contract(
     panic!("Failed to create contract")
 }
 
+pub fn compile_solidity_contract(name: &str, solidity_src: &str) -> Result<(String, String), ()> {
```

Comment:
@satyamakgec This is all just WIP code in this PR but I wanted to make you aware of it because it adds a way to run tests against solidity code (which I needed to prove some assumptions about revert reason encoding). Since the differential contract testing will also need to run solidity code I thought you might find it useful to look at.

Reply:
You are a saviour @cburgdorf I was worried about this today when I was doing researching and planning over the differential testing.

Reply:
Very useful tools!